src/app/auth/services/auth.service.ts

```diff
@@ -90,6 +93,7 @@ export type AuthenticateUserParams = {
   token?: string;
   redeemCodeObject?: boolean;
   doSignUp?: RegisterFunction;
+  opaqueLogin?: boolean;
```

Comment:
change name of this boolan variablef for `useOpaqueLogin` or something similar that matches the purpose.